build_tools/python/e2e_test_framework/models/model_groups.py

```diff
@@ -9,20 +9,15 @@
 
 # Small models that require less computational resources.
 SMALL = [
-    tflite_models.DEEPLABV3_FP32,
-    tflite_models.EFFICIENTNET_INT8,
-    tflite_models.MOBILEBERT_FP16,
-    tflite_models.MOBILEBERT_FP32,
-    tflite_models.MOBILEBERT_INT8,
-    tflite_models.MOBILENET_V1,
-    tflite_models.MOBILENET_V2,
-    tflite_models.MOBILENET_V2_INT8,
-    tflite_models.MOBILENET_V3SMALL,
-    tflite_models.MOBILESSD_FP32,
-    tflite_models.PERSON_DETECT_INT8,
-    tflite_models.POSENET_FP32,
+    tflite_models.DEEPLABV3_FP32, tflite_models.EFFICIENTNET_INT8,
```

Comment:
Can you revert the formatting change here?

Reply:
Reverted